.github/actions/vaas-scan-action/README.md

```diff
@@ -0,0 +1,59 @@
+# vaas-scan-action
+
+This is a github action with that one can scan changes in pull-requests with the G DATA product Verdict-as-a-Service (VaaS). VaaS is an Antivirus in the cloud, that can be utilized by a simple api.
+
+## preconditions
+
+Before you can use this action, you need to aquire credentials. Please find more Information on the [Verdict-as-a-Service product page](https://www.gdata.de/business/security-services/verdict-as-a-service). If you want to test VaaS on your own, you can get credentials on our [registration page](https://vaas.gdata.de/login)
```

Comment:
Shouldn't we link to the onboarding guide at https://vaas.gdata.de first? That guide contains the referral to `/login`.

Reply:
yeah, had that in first, dunno, just gimme your suggestions

Reply:
I would prefer the guide, it's makes it clearer what to do when I click on that link.

Reply:
You are right. Will change that tomorrow, when nobody has a different opinion.